Detectors/TOF/reconstruction/src/Clusterer.cxx

```diff
@@ -100,6 +100,20 @@ void Clusterer::addContributingDigit(Digit* dig)
 
   if (mNumberOfContributingDigits == 6) {
     LOG(ERROR) << "The cluster has already 6 digits associated to it, we cannot add more; returning without doing anything";
+
```

Comment:
I assume this should fix the crash I had in my test. So, you add here a protection for the array overflow, but how one can have at all > 6 neighbours?

Reply:
Hello Ruben, 
We checked, and we think that a better tuned clusterization could be used. I mean, the >6 digits that go to the same cluster could actually become 3 clusters if we applied a different cut on the time difference between digits (now at 500 ps). We plan to study this in more details, but for now, we decided to leave the cut as it is. 
Cheers,
Chiara

Reply:
OK, thanks. For the failing o2checkcode/o2 test: this is a general problem, for some reason o2sim is not found in the build env, @dberzano was notified.

Cheers,
 Ruben